lib/BackgroundJob/Tasks/CreateClustersTask.php

```diff
@@ -159,6 +160,10 @@ private function createClusterIfNeeded(string $userId) {
 				$this->logInfo('Clusters already exist, but there was some change that requires recreating the clusters');
 			}
 		} else {
+			// User should not be able to use this directly, used in tests
+			$forceCreateClusters = $this->config->getUserValue($userId, 'facerecognition', 'force-create-clusters', 'false');
+			$forceCreation = ($forceCreateClusters === 'true');
```

Comment:
I do not like to add this, but I guess it's acceptable ..

Reply:
yeah, it is screaming "this code is added because of tests":) Now, that I am doing stress tests, I will read more about mocking capabilities of PHPUnit, so I might cook something with mocks (instead of configs like this).

Reply:
Wow.. Ok..